.travis.yml

```diff
@@ -21,48 +28,39 @@ jobs:
         - NODE=$(which node) make lint lint-py
 
     - name: "Compile V8"
-      language: cpp
       cache: ccache
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test
           packages:
             - g++-6
-      install:
-        - CC='ccache gcc-6' CXX='ccache g++-6' ./configure
+      install: *ccache-setup-steps
       script:
-        - CC='ccache gcc-6' CXX='ccache g++-6' make -j2 -C out V=1 v8
+        - ./configure
+        - make -j2 -C out V=1 v8
 
     - name: "Compile Node.js"
-      language: cpp
       cache: ccache
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test
           packages:
             - g++-6
-      install:
-        - CC='ccache gcc-6' CXX='ccache g++-6' ./configure
+      install: *ccache-setup-steps
       script:
-        - CC='ccache gcc-6' CXX='ccache g++-6' make -j2 V=1
+        - ./configure
+        - make -j2 V=1
+        - make -j1 V=1 build-addons build-js-native-api-tests build-node-api-tests
+        - python tools/test.py -j 2 -p dots --mode=release --flaky-tests=dontcare addons js-native-api node-api
```

Comment:
I'd prefer not to run tests in a job called "Compile". We could have parallel jobs in the Tests stage if you don't want to run all the suites in one job.

Reply:
Yeah. That makes sense.
I wanted to save the GCC-6 bootstrapping time. But we adding a job that only does the native addons would make sense. Just need to figure out what else it needs.

Reply:
> 
> 
> Yeah. That makes sense.
> I wanted to save the GCC-6 bootstrapping time. But we adding a job that only does the native addons would make sense. Just need to figure out what else it needs.

I think the arm-fanned jobs on our CI would be a good starting point.

Reply:
Done